### DISCUSSION TITLE
Can't resolve '<'

### DISCUSSION THREAD

**Discussion by matthias-ccri** [facebook/docusaurus#9842] [General] [answered]
When running docusaurus build, it give me:

```
Module not found: Error: Can't resolve '<' in '/home/user/src/repo/api-docs-site/docs'
[ERROR] Client bundle compiled with errors therefore further build is impossible.
```

Why doesn't it show a stack trace or which file has the error? Instead it points to a folder "docs" which in my case has about 500 files in it.

Is it possible for docusaurus to improve this error message? Are there verbose flags? Or any way I can debug this besides shooting in the dark?

**Comment by slorber** [comment]
Hey

I'd like to help you but it's impossible without a repro and without knowing the version of Docusaurus you are using.

This error is probably new so if you checkout the last commits to could probably find an older version that works, and isolate a change that lead to this new error. THen you can share the diff with me.

**Comment by matthias-ccri** [comment, marked as answer]
Thanks for replying. I was using an older version of docusaurus. The latest version shows a more detailed error with filenames and line and column numbers.

I'll close this
